scripts/verify.js

```diff
@@ -0,0 +1,78 @@
+// We require the Hardhat Runtime Environment explicitly here. This is optional
+// but useful for running the script in a standalone fashion through `node <script>`.
+//
+// You can also run a script with `npx hardhat run <script>`. If you do that, Hardhat
+// will compile your contracts, add the Hardhat Runtime Environment's members to the
+// global scope, and execute the script.
+const { exec } = require("child_process");
+const hre = require("hardhat");
```

Comment:
can be removed

Reply:
I agree on the `just deploy-verify SOURCE DESTINATION` recipe. It will also simplify things on the Foundry side. I'll implement both in a separate PR. 👍